ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/misc.py

```diff
@@ -262,3 +262,7 @@ def fabric_sflow_mlag_interfaces(self: SharedUtils) -> bool | None:
     def default_interface_mtu(self: SharedUtils) -> int | None:
         default_default_interface_mtu = get(self.hostvars, "default_interface_mtu")
         return get(self.platform_settings, "default_interface_mtu", default=default_default_interface_mtu)
+
+    @cached_property
+    def evpn_multicast(self) -> bool:
+        return get(self.hostvars, "switch.evpn_multicast") is True
```

Comment:
you cannot move this to shared_utils since you check for switch.*. Instead this should point to whatever the evpn_multicast fact point to.

Reply:
that's bad news https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts/eos_designs_facts.py#L90

Reply:
sounds like we have an issue then..